lib/Controller/MailboxesController.php

```diff
@@ -226,11 +226,7 @@ public function markAllAsRead(int $id): JSONResponse {
 	 */
 	public function stats(int $id): JSONResponse {
 		$mailbox = $this->mailManager->getMailbox($this->currentUserId, $id);
-		$account = $this->accountService->find($this->currentUserId, $mailbox->getAccountId());
-
-		$stats = $this->mailManager->getMailboxStats($account, $mailbox);
-
-		return new JSONResponse($stats);
+		return new JSONResponse($mailbox->getFolderStats());
```

Comment:
that means we'll always use the cached version here, right?

Reply:
Exactly. AFAIK, it has only been used for the mailbox context menu in the navigation pane until now.

Reply:
I highly agree. Wasn't sure if the refactor is appropriate here but I can implement it.